packages/connectivity/src/scp-cf/authorization-header.ts

```diff
@@ -164,14 +184,12 @@ interface AuthenticationHeaders {
 
 function getProxyRelatedAuthHeaders(
   destination: Destination
-): AuthenticationHeaderProxy | undefined {
+): AuthenticationHeaderProxy | AuthenticationNoAuthHeaders | undefined {
```

Comment:
[q] I think this defeats the purpose of this function. There are two functions for proxy related headers and auth related headers. I think this might be intended differently than you "hacked" it ;)